package.json

```diff
@@ -155,16 +155,13 @@
         "tsconfig-paths": "4.1.0",
         "typedoc": "0.25.3",
         "typedoc-plugin-merge-modules": "5.1.0",
-        "typescript": "5.6.3",
+        "typescript": "5.8.2",
         "typescript-eslint": "^8.22.0",
         "wait-on": "6.0.1",
         "wdio-axe": "2.2.3",
         "wdio-chromedriver-service": "6.0.4",
         "webdriverio": "7.27.0"
     },
-    "resolutions": {
```

Comment:
we need to keep that section, otherwise nx@npm gets multiple versions installed and CI fails. Maybe you just update it with 20.6.2

Reply:
but the pipeline has passed

Reply:
it wasn't failing all the time
before either

Reply:
can we merge it like that and ping me if it fails again, I want to investigate it and find why

Reply:
I took a look at the yarn.lock file and saw the nx was mentioned twice
https://github.com/SAP/fundamental-ngx/blob/d27e32708c4784785ac74bd51d83db0f121a28b6/yarn.lock
at lines 19202 and 19268
but at this PR nx is mentioned only once, so there shouldn't be a problem now, I'm not sure what caused the other one to have it twice and it will be useless now to investigate
if this happened again in the main branch I will investigate and resolve it

Reply:
why people in angular 19.0 and 19.1 won't be able to use if we merged this PR
angular shouldn't have any breaking changes between minor versions, so a library built with 19.2 should be fully compatible with 19.0 and 19.1

Reply:
I created the PR and I wasn't sure if I could solve all the problems that's why I created it draft so I don't waste someone's time in reviewing it because there were multiple issues with typescript
I will mark it ready now